main.go

```diff
@@ -40,6 +40,10 @@ var (
 	includeServerResourceID = flag.Bool("include-server-resource-id", false, "whether or not to generate config for server resource id. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 )
 
+// The `id` field of the Node proto is set to a value of the format
+// `projects/{project number}/networks/{network name}/nodes/{uuid}`.
+const nodeIDFormatStr = "projects/%d/networks/%s/nodes/%s"
```

Comment:
Inline. Splitting a format string from its arguments is asking for trouble.

Reply:
Done.